src/test/java/com/org/jenkins/custom/jenkins/distribution/service/PluginControllerTest.java

```diff
@@ -33,6 +36,7 @@ public void return_status_200() throws Exception {
         String uri = "/api/plugin/getPluginList";
         MvcResult mvcResult = mvc.perform(MockMvcRequestBuilders.get(uri).accept(MediaType.APPLICATION_JSON_VALUE)).andReturn();
         int status = mvcResult.getResponse().getStatus();
+        assertThat(new JSONObject(dummyUpdateBody).toString(), CoreMatchers.containsString(mvcResult.getResponse().getContentAsString()));
```

Comment:
If the map is removed this body returns null so I decided to check the contents of the response and match the dummy update body.